Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListCollectionViewCell.swift

```diff
@@ -5,4 +5,62 @@
 import Foundation
 import UIKit
 
-open class ChatChannelListCollectionViewCell: UICollectionViewCell {}
+open class ChatChannelListCollectionViewCell<ExtraData: UIExtraDataTypes>: UICollectionViewCell {
+    // MARK: - Properties
+    
+    public weak var collectionView: UICollectionView?
+    
+    public var channelView: ChatChannelView<ExtraData>? {
+        didSet {
+            oldValue?.removeFromSuperview()
+            if let new = channelView {
+                contentView.embed(new)
+            }
+        }
+    }
+    
+    // MARK: - Init
+    
+    override public init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+
+    public required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+
+    private func commonInit() {
+        contentView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            contentView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            contentView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            contentView.topAnchor.constraint(equalTo: topAnchor),
+            contentView.bottomAnchor.constraint(equalTo: bottomAnchor)
```

Comment:
Hah, that's interesting!

`contentView` uses autoresizing-mask to have the same size with the cell. What I tried to do here is to get rid of the mask approach (by `contentView.translatesAutoresizingMaskIntoConstraints = false`) and setup the constraints manually. But it makes no sense since by default `content view translates its autoresizing-mask into constraints` 😄